docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:9.13-slim
+FROM debian:10.9-slim
```

Comment:
Can we run this through a soak test to ensure that resource usage is not affected?

Reply:
Done. See b/186568780#comment11

Reply:
The RSS usage looks like it's flattening out after ~18h:

![image](https://user-images.githubusercontent.com/7594381/116730645-bc1b9b80-a9b6-11eb-885f-836e132094a1.png)

So we can get this in and let it run over 4 days.

Reply:
The RSS usage looks like it's flattening out after ~18h:

![image](https://user-images.githubusercontent.com/7594381/116730767-e1100e80-a9b6-11eb-8c3c-68c0ac47ce60.png)

So we can get this in and let it soak over 4d.